### DISCUSSION TITLE
[Feature/Hardware Request] ATI Radeon 7500

### DISCUSSION THREAD

**Discussion by mrmaller1905** [86Box/86Box#4810] [Ideas]
Can you add support for emulation of ATI Radeon 7500 or 7200 DDR to 86Box?

Thanks in advance.

**Comment by MrPepka** [comment]
No

**Comment by jriwanek** [comment]
As per every other time people ask us for higher end cards, the documentation available is slim to none, they'd be rough to emulate on even more recent machines, and therefore arn't going to happen.

**Comment by ppgrainbow** [comment]
Not going to happen, now or in the near future. There is little to known documentation on any ATI Radeon cards, the graphics cards have either 32 MB or 64 MB VRAM, making it nearly impossible to emulate.
